public/app/features/datasources/partials/tls_auth_settings.html

```diff
@@ -7,17 +7,23 @@ <h6>TLS/SSL Auth Details</h6>
     <div class="gf-form-inline">
       <div class="gf-form gf-form--v-stretch"><label class="gf-form-label width-7">CA Cert</label></div>
       <div class="gf-form gf-form--grow" ng-if="!current.secureJsonFields.tlsCACert">
-         <textarea
-           rows="7"
-           class="gf-form-input gf-form-textarea"
-           ng-model="current.secureJsonData.tlsCACert"
-           placeholder="Begins with -----BEGIN CERTIFICATE-----"
-         ></textarea>
+        <textarea
+          rows="7"
+          class="gf-form-input gf-form-textarea"
+          ng-model="current.secureJsonData.tlsCACert"
+          placeholder="Begins with -----BEGIN CERTIFICATE-----"
+        ></textarea>
       </div>
 
       <div class="gf-form" ng-if="current.secureJsonFields.tlsCACert">
         <input type="text" class="gf-form-input max-width-12" disabled="disabled" value="configured" />
-        <a class="btn btn-secondary gf-form-btn" href="#" ng-click="current.secureJsonFields.tlsCACert = false">reset</a>
+        <button
+          type="reset"
+          class="btn btn-secondary gf-form-btn"
+          ng-click="current.secureJsonFields.tlsCACert = false"
+        >
+          reset
```

Comment:
We should use a better label for the buttons, like `Reset Client Cert`or use a descriptive aria-label.

Reply:
added aria labels